packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -221,18 +243,69 @@ void main() {
     });
   });
 
+  group('host validator', () {
```

Comment:
Place these tests in a host_validator_test.dart file instead of adding to the doctor_test.dart